examples/wannier.jl

```diff
@@ -0,0 +1,161 @@
+# keyword argument `fileprefix` as shown below.
+#
+# We now produce a simple Wannier model for 5 MLFWs.
+#
+# For a good MLWF, we need to provide initial projections that resemble the expected shape
+# of the Wannier functions.
+# Here we will use:
+# - 3 bond-centered 2s hydrogenic orbitals for the expected σ bonds
+# - 2 atom-centered 2pz hydrogenic orbitals for the expected π bands
+
+using Wannier # Needed to make Wannier.Model available
+
+# From chemical intuition, we know that the bonds with the lowest energy are:
+# - the 3 σ bonds,
+# - the π and π* bonds.
+# We provide relevant initial projections to help Wannierization
+# converge to functions with a similar shape.
+s_guess(center) = DFTK.HydrogenicWannierProjection(center, 2, 0, 0, C.Z)
+pz_guess(center) = DFTK.HydrogenicWannierProjection(center, 2, 1, 0, C.Z)
+projections = [
+    ## Note: fractional coordinates for the centers!
+    ## 3 bond-centered 2s hydrogenic orbitals to imitate σ bonds
+    s_guess((positions[1] + positions[2]) / 2),
+    s_guess((positions[1] + positions[2] + [0, -1, 0]) / 2),
+    s_guess((positions[1] + positions[2] + [-1, -1, 0]) / 2),
+    ## 2 atom-centered 2pz hydrogenic orbitals
+    pz_guess(positions[1]),
+    pz_guess(positions[2]),
+]
+
+# Wannierize:
+wannier_model = Wannier.Model(scfres;
+    fileprefix="wannier/graphene",
+    n_bands=scfres.n_bands_converge,
+    n_wannier=5,
+    projections,
+    dis_froz_max=ustrip(auconvert(u"eV", scfres.εF))+1) # maximum frozen window, for example 1 eV above Fermi level
+
+# Once we have the `wannier_model`, we can use the functions in the Wannier.jl package:
+#
+# Compute MLWF:
+U = disentangle(wannier_model, max_iter=200);
+
+# Inspect localization before and after Wannierization:
+omega(wannier_model)
+omega(wannier_model, U)
+
+# Build a Wannier interpolation model:
+kpath = irrfbz_path(model)
+interp_model = Wannier.InterpModel(wannier_model; kpath=kpath)
+
+# And so on...
+# Refer to the Wannier.jl documentation for further examples.
+
+# (Delete temporary files when done.)
+rm("wannier", recursive=true)
+
+# ### Custom initial guesses
+#
+# We can also provide custom initial guesses for Wannierization,
+# by passing a callable function in the `projections` array.
+# The function receives the basis and a list of points (fractional coordinates in reciprocal space),
+# and returns the Fourier transform of the initial guess function evaluated at each point.
+#
+# For example, we could use Gaussians for the σ and pz guesses with the following code:
+s_guess(center) = DFTK.GaussianWannierProjection(center)
+pz_guess(center) = begin
+    ## Approximate with two Gaussians offset by 0.5 Å from the center of the atom
+    offset = model.inv_lattice * [0, 0, austrip(0.5u"Å")]
+    center1 = center + offset
+    center2 = center - offset
+    ## Build the custom projector
+    (basis, qs) -> DFTK.GaussianWannierProjection(center1)(basis, qs) - DFTK.GaussianWannierProjection(center2)(basis, qs)
+end
```

Comment:
Actually code up thd call. At least in a comment

Reply:
not sure I understand

Reply:
That's fine no worries, not too important anyways.